Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2852,16 +2854,17 @@ extension BrowserViewController {
          Specific values here are from the WKContentView headers.
          https://github.com/JaviSoto/iOS9-Runtime-Headers/blob/master/Frameworks/WebKit.framework/WKContentView.h
         */
-        guard let webContentView = UIView.findSubViewWithFirstResponder(webView),
-            let input = webContentView.performSelector(Selector("inputAccessoryView")),
+        guard let webContentView = UIView.findSubViewWithFirstResponder(webView) else {
+            return
```

Comment:
Note that assertions are only run in developer builds, not release builds. It's good practice to use asserts aggressively since they quickly reveal bugs, with stack traces to boot, and have no effect on the released product.